docs/developer-docs/smart-contracts/maintain/upgrade.mdx

```diff
@@ -11,28 +11,21 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
 ## Overview
 
-Unlike a <GlossaryTooltip>canister</GlossaryTooltip> reinstall that preserves the canister identifier but no state, a canister upgrade enables you to preserve the state of a deployed canister, and change the code.
+Unlike a <GlossaryTooltip>canister</GlossaryTooltip> reinstall that preserves the canister identifier but no state, a canister upgrade enables you to preserve the state of a deployed canister and change the code.
 
 For example, assume you have a dapp that manages professional profiles and social connections. If you want to add a new feature to the dapp, you need to be able to update the canister code without losing any of the previously-stored data. A canister upgrade enables you to update existing canister identifiers with program changes without losing the program state.
 
+:::tip
 To preserve state when you are upgrading a canister written in Motoko, be sure to use the `stable` keyword to identify the variables you want to preserve. For more information about preserving variable state in Motoko, see [stable variables and upgrade methods](/docs/current/motoko/main/canister-maintenance/upgrades).
+:::
 
-To upgrade a canister:
+To upgrade a canister, first you must:
 
-- #### Step 1:  Open a new terminal and navigate to your project directory.
+1. Edit the canister's code and save the changes.
```

Comment:
Should you specify that you should save the changes in the IDE? Currently, it reads that you must use a command to save the changes.

Reply:
I think "Modify the canister's code." would be enough. should be clear that changes must be saved to build a new version 😅